code/controllers/subsystem/explosions.dm

```diff
@@ -700,14 +710,15 @@ SUBSYSTEM_DEF(explosions)
 			if (length(details) != 3)
 				continue
 			var/throw_range = details[1]
-			var/throw_dir = details[2]
+			var/turf/center = details[2]
 			var/max_range = details[3]
 			for(var/atom/movable/A in explode)
 				if(QDELETED(A))
 					continue
 				if(!A.anchored && A.move_resist != INFINITY)
-					var/atom_throw_range = rand(throw_range, max_range)
-					var/turf/throw_at = get_ranged_target_turf(A, throw_dir, atom_throw_range)
+					// We want to have our distance matter, but we do want to bias to a lot of throw, for the vibe
+					var/atom_throw_range = rand(throw_range, max_range) + max_range * 0.3
+					var/turf/throw_at = get_ranged_target_turf_direct(A, center, atom_throw_range, 180) // Throw 180 degrees away from the explosion source
```

Comment:
oh also I've made it use an angled system rather then dir, so it'll be more of a circle. forgot to mention that